packages/flutter_tools/lib/src/ios/mac.dart

```diff
@@ -251,13 +251,26 @@ Future<XcodeBuildResult> buildXcodeProject({
   );
 
   if (hasPlugins()) {
-    final String currentGeneratedXcconfig = readGeneratedXcconfig(app.appDirectory);
-    await cocoaPods.processPods(
+    final String iosPath = fs.path.join(fs.currentDirectory.path, app.appDirectory);
+    // If the Xcode project, Podfile, or Generated.xcconfig have changed since
+    // last run, pods should be updated.
+    final Fingerprinter fingerprinter = new Fingerprinter(
+      fingerprintPath: fs.path.join(getIosBuildDirectory(), 'pod_inputs.fingerprint'),
```

Comment:
The fingerprint file should probably be .gitignored in our project templates. Maybe just add a `*.fingerprint` pattern at the root, and have the `Fingerprinter` class help by enforcing the use of the `.fingerprint` extension.

Reply:
Nevermind. I see now that the fingerprint file has been placed in the build folder.